packages/fluentui/docs/src/prototypes/Calls/recentContact.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import { Avatar, Flex, Text, Menu } from '@fluentui/react-northstar';
+import { CallIcon, CallVideoIcon } from '@fluentui/react-icons-northstar';
+
+const Contact: React.FC<{
+  image?: string;
+  name: string;
+  status: object;
+}> = ({ image, name, status }) => {
+  return (
+    <Flex fill gap="gap.small" space="between">
+      <div>
```

Comment:
I agree that using `<div />` is the correct thing, but this is not reflecting 100% the test case we want to create. Let's use `<Box />` or `<Flex />` here. Later if we want to show how big improvements changes like this can make, it will be more obvious.